catalog/async/catalog-async-data/src/main/java/org/codice/ddf/catalog/async/data/impl/LazyProcessResourceImpl.java

```diff
@@ -86,7 +89,16 @@ public LazyProcessResourceImpl(String metacardId, Supplier<Resource> resourceSup
   @Override
   public InputStream getInputStream() throws IOException {
     loadResource();
-    return inputStream;
+    if (inputStreamDataCache == null) {
+      if (inputStream == null) {
+        throw new IOException(String.format("Tried to get input stream for %s but was null", name));
+      }
+      inputStreamDataCache = new TemporaryFileBackedOutputStream(32 * 1024);
```

Comment:
✏️ Why 32K? How did you come up with that? Should define this limit as a static constant.

Reply:
I followed the suit of the only current post processing plugin that I know of https://github.com/codice/alliance/pull/559/files#diff-a06efeb9a29c435285cd055da3d3fec7R128. It's doing something similar to what I'm doing here. I used that because I know it works and using the TemporaryFileBackedOutputStream default of 1M causes out of memory errors.

Reply:
I knew too big would be an issue. I was just wondering where the 32K came from. Please define this as a constant so you can comment on it.